no-browser-site-scrapers/Albertsons/index.js

```diff
@@ -0,0 +1,44 @@
+const { site } = require("./config");
+const { toTitleCase } = require("../../lib/stringUtil");
+const https = require("https");
+const { sendSlackMsg } = require("../../lib/slack");
+
+module.exports = async function GetAvailableAppointments() {
+    console.log(`${site.name} starting.`);
+    const webData = await ScrapeWebsiteData();
+    console.log(`${site.name} done.`);
+
+    const massLocations = webData.filter(
+        (location) => location.region == "Massachusetts"
+    );
+    return massLocations.map((location) => {
+        const retval = {
+            name: `${site.name} (${location.address})`,
+            hasAvailability: location.availability == "true",
+            signUpLink: location.coach_url,
+            latitude: location.lat,
+            longitude: location.long,
+        };
+        return retval;
+    });
+};
+
+function urlContent(url, options) {
+    return new Promise((resolve) => {
+        let response = "";
+        https.get(url, options, (res) => {
```

Comment:
Nitpick but we do include the `node-fetch` library and use it in other places rather than having to build the request in vanilla node.

Reply:
Ah, thanks. I was cribbing from another scraper (CVSPharmacy) that used this approach. I'll look into the `node-fetch` method. Thanks for reviewing the patch!

Reply:
Yes, that was my fault and I regret it. Sorry for leading you astray.

Reply:
@johnhawkinson No worries!